.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.3.0
```

Comment:
Given that this gem is intended to work with a wide range of ruby versions, we should probably exclude this file.